lib/node_modules/@stdlib/math/base/special/coth/LICENSE

```diff
@@ -0,0 +1,195 @@
+                                Apache License
```

Comment:
please revert this

Reply:
uh.. I should leave an empty line at the start of the file right?